pkg/rsnotify/listenerfactory/listenerfactory.go

```diff
@@ -0,0 +1,27 @@
+package listenerfactory
```

Comment:
The code here was moved from `factory/listenerfactory.go`. None of it has changed, apart from removing the implementations and moving them to the correct listener implementation packages.